### DISCUSSION TITLE
steam: Steam now requires user namespaces to be enabled (userns)

### DISCUSSION THREAD

**Discussion by CaseOf** [netblue30/firejail#6439] [Q&A]
Hello,
I am trying to run Steam under firejail with this `steam.local`:
```
ignore mkdir
ignore mkfile

whitelist ${HOME}

private ${HOME}/.local/steam

private-bin awk,basename,bash,bsdtar,bzip2,cat,chmod,cksum,cmp,comm,compress,cp,curl,cut,date,dbus-launch,dbus-send,desktop-file-edit,desktop-file-install,desktop-file-validate,dirname,echo,env,expr,file,find,getopt,grep,gtar,gzip,head,hostname,id,lbzip2,ldconfig,ldd,ln,ls,lsb_release,lsof,lspci,lz4,lzip,lzma,lzop,md5sum,mkdir,mktemp,mv,netstat,ps,pulseaudio,python*,readlink,realpath,rm,sed,sh,sha1sum,sha256sum,sha512sum,sleep,sort,steam,steamdeps,steam-native,steam-runtime,sum,tail,tar,tclsh,test,touch,tr,umask,uname,update-desktop-database,wc,wget,wget2,which,whoami,xterm,xz,zenity

```

and I am actually getting this error from Steam:
```
Steam now requires user namespaces to be enabled.

This requirement is the same as for Flatpak, which has more detailed
information available:
https://github.com/flatpak/flatpak/wiki/User-namespace-requirements
```
User namespaces are loaded on my kernel.
What could I do to allow Steam to use user namespaces?
It is heavily used by Steam to start games via Proton.

**Comment by rusty-snake** [comment]
1. Basic debugging informations are missing.

> whitelist ${HOME}
private ${HOME}/.local/steam

Does not look healthy.

> User namespaces are loaded on my kernel.

And enabled, for unprivileged users?

**Comment by CaseOf** [comment]
Here is debug log: https://gist.github.com/CaseOf/0adbf3e655a7d2648b3916d183b25cfe
After some research, it seems to be from its unability to use bubblewrap : https://gitlab.steamos.cloud/steamrt/steam-runtime-tools/-/blob/main/bin/check-requirements.c#L134

**Comment by CaseOf** [comment]
Ok, it seems to be private-bin to be blocking it, bwrap not being listed.

**Comment by CaseOf** [comment]
works outside firejail else it crashes actually with a window giving choices inside firejail
